docs/examples/droplet_fusion/droplet_fusion.rst

```diff
@@ -0,0 +1,252 @@
+    plt.ylabel(r"x-coordinate ($\mu$m)")
+    plt.tight_layout()
+    plt.show()
+
+.. image:: selected_data.png
+
+Model for fusion
+^^^^^^^^^^^^^^^^
+
+The force data during the fusion event is fitted with the following equation: :math:`f(t) = ae^{-t/\tau}+bt+c`
+
+The term :math:`bt` accounts for the movement of the trap, assuming a constant trap speed [1]_. 
+(When both traps are static, the term :math:`bt` should be removed from the model.)
+The parameter of interest is :math:`𝜏`, the relaxation time scale of the fusion event::
+
+    from scipy.optimize import curve_fit
+
+    def relaxation_model(t, tau, a, b, c):
+        return a * np.exp(-t / tau) + b * t + c
+
+Fit the data and plot the result::
+
+    time = force_selection.seconds
+    force = force_selection.data
+
+    popt, pcov = curve_fit(relaxation_model, time, force, [0.1, force[0], 0, 0])
+    plt.figure()
+    plt.plot(time, force)
+    plt.plot(time, relaxation_model(time,*popt), label=fr"$\tau$ = {popt[0]:0.2f}s")
+    plt.ylabel(r"x-coordinate ($\mu$m)")
+    plt.xlabel("Time (s)")
+    plt.legend()
+    plt.show()
+
+.. image:: fit.png
+
+The array :math:`popt` contains all the fitted parameters::
+
+    >>>> print(popt)
+
+    [ 0.04754597  5.39347958 -4.5188036  -6.2716303 ]
+
+The first parameter in :math:`popt` is :math:`𝜏` and the other 3 parameters are :math:`a`, :math:`b` and :math:`c` respectively, as defined in the model above.
+The matrix :math:`pcov` is the covariance matrix and the standard deviation errors in the fitted parameters can be obtained as:: 
+
+    >>>> np.sqrt(np.diag(pcov))
+
+    [0.00021354, 0.01104559, 0.0218276 , 0.00779505]
+
+The relaxation time obtained from the fit is 0.0475 +- 0.0002 seconds. 
+
+Now, we will proceed to determine the size of the droplets before the fusion event.
+
+Droplet size
+------------
+
+First load the scan and print the relevant metadata::
+
+    >>> for name, scan in f.scans.items():
+    >>>        print(f"num frames: {scan.num_frames}")
+    >>>        frame_duration = (scan.frame_timestamp_ranges()[0][1]-scan.frame_timestamp_ranges()[0][0])/1e9
+    >>>        print(f"frame duration: {frame_duration} s")
+
+    num frames: 27
+    frame duration: 0.4977664 s
+
+Plot a frame before the fusion event::
+
+    framenr = 2
+    plt.figure()
+    scan.plot(channel="green", frame=framenr)
+    plt.show()
+
+.. image:: frame2.png
+
+Plot a frame after the fusion event::
+
+    framenr = 6
+    plt.figure()
+    scan.plot(channel="green", frame=framenr)
+    plt.show()
+
+.. image:: frame6.png
+
+If the droplets are in focus, the size of the droplet can be estimated from the 2D scan. 
+The estimate will not be extremely precise, because the boarders of the scanned images are not sharp.
+The estimate in the size will therefore have an error of ~10%.
+
+The first step, is to use image segmentation to identify the two droplets in the image. 
+The threshold may need to be optimized for your data::
+
+    import math
```

Comment:
Do you actually use this import? If not, should probably remove it.

Reply:
it is not used anymore in the current workflow, removed it